services/dynamic-sidecar/tests/unit/test_modules_seven_zip_wrapper.py

```diff
@@ -0,0 +1,146 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+
+import subprocess
+from pathlib import Path
+
+import pytest
+from _pytest._py.path import LocalPath
+from faker import Faker
+from models_library.basic_types import IDStr
+from models_library.progress_bar import ProgressReport
+from servicelib.archiving_utils import archive_dir, unarchive_dir
+from servicelib.progress_bar import ProgressBarData
+from simcore_service_dynamic_sidecar.modules.seven_zip_wrapper import (
+    SevenZipError,
+    unarchive_zip_to,
+)
+
+
+def _ensure_path(dir_path: Path) -> Path:
+    dir_path.mkdir(parents=True, exist_ok=True)
+    return dir_path
+
+
+def _assert_same_directory_content(path1: Path, path2: Path) -> None:
+    assert path1.is_dir()
+    assert path2.is_dir()
+
+    contents1 = {p.relative_to(path1) for p in path1.rglob("*")}
+    contents2 = {p.relative_to(path2) for p in path2.rglob("*")}
+
+    assert contents1 == contents2
+
+
+@pytest.fixture
+def to_archive_dir(tmpdir: LocalPath) -> Path:
```

Comment:
This is like a deja vu from to year one's reviews  :-) 

We use everywhere the `tmp_path` version of this fixture which produces `pathlib.Path` instead of `tmpdir` which produces a custom version of pytest `LocalPath`

Please read https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture  and use `tmp_path: Path` instead of `tmpdir: LocalPath`